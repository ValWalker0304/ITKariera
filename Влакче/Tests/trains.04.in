Add 54665 Shumen F 4
Add 61307 Blagoevgrad P 8
Add 33896 Burgas P 3
Add 61385 Sofia P 4
Add 93392 Svilengrad P 6
Add 13725 Burgas F 37
Add 27471 Varna P 7
Add 85568 Svilengrad F 8
Add 51340 Zimnitsa F 24
Add 23931 Blagoevgrad F 16
Travel
Travel
Add 88311 Podkova F 13
Next
Next
Add 52372 Sindel P 1
Add 21480 Svilengrad P 6
End