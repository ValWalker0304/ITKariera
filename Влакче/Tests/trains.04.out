23931 Blagoevgrad F 16
51340 Zimnitsa F 24
27471 Varna P 7
27471 Varna P 7
